corehq/apps/export/models/deid_export.py

```diff
@@ -0,0 +1,42 @@
+import hashlib
+import uuid
+
+from django.db import models
+
+from corehq.const import ONE_DAY
+from corehq.util.quickcache import quickcache
+
+
+class DeIdMapping(models.Model):
+    domain = models.TextField(max_length=255)
+    hashed_value = models.TextField(max_length=32)
+    deid = models.UUIDField(default=uuid.uuid4)
```

Comment:
If we do stick with the db model, could we just make the primary key a uuid and use that instead of the deid field? (Code might not be 100% accurate)
`id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)`

Reply:
+1